README.md

```diff
@@ -1,51 +1,70 @@
+# Metabase
+Metabase is the easy, open source way for everyone in your company to ask questions and learn from data.
+
+![Metabase Product Screenshot](docs/metabase-product-screenshot.png)
 [![Latest Release](https://img.shields.io/github/release/metabase/metabase.svg?label=latest%20release)](https://github.com/metabase/metabase/releases)
 [![GitHub license](https://img.shields.io/badge/license-AGPL-05B8CC.svg)](https://raw.githubusercontent.com/metabase/metabase/master/LICENSE.txt)
 [![Circle CI](https://circleci.com/gh/metabase/metabase.svg?style=svg&circle-token=3ccf0aa841028af027f2ac9e8df17ce603e90ef9)](https://circleci.com/gh/metabase/metabase)
 [![Leiningen Dependencies Status](https://jarkeeper.com/metabase/metabase/status.svg)](https://jarkeeper.com/metabase/metabase)
 [![NPM Dependencies Status](https://david-dm.org/metabase/metabase.svg)](https://david-dm.org/metabase/metabase)
 
-# Overview
-
-Metabase is an easy way to generate charts and dashboards, ask simple ad hoc queries without using SQL, and see detailed information about rows in your Database. You can set it up in under 5 minutes, and then give yourself and others a place to ask simple questions and understand the data your application is generating. It is not tied to any specific framework and can be used out of the box with minimal configuration.
-
-With a bit of tagging and annotation of what the tables and fields in your database mean, it can be used to provide a rich, humanized version analytics server and administration interface.
+# Features
+- 5 minute [setup](http://www.metabase.com/docs/latest/setting-up-metabase) (We're not kidding)
+- Let anyone on your team [ask questions](http://www.metabase.com/docs/latest/users-guide/03-asking-questions) without knowing SQL
+- Rich beautiful [dashboards](http://www.metabase.com/docs/latest/users-guide/05-sharing-answers) with auto refresh and fullscreen
+- SQL Mode for analysts and data pros.
+- Create canonical [segments and metrics](http://www.metabase.com/docs/latest/administration-guide/05-segments-and-metrics) for your team to use.
+- Send data to Slack or email on a schedule with [Pulses](http://www.metabase.com/docs/latest/users-guide/07-pulses)
```

Comment:
Nitpicky, but for visual consistency I think it's preferable to either omit punctuation in bullet lists or don't, rather than mixing the two (i.e., the period at the end of this bullet point makes me :crying_cat_face: )